test-flip-link-app/examples/panic.rs

```diff
@@ -0,0 +1,15 @@
+#![no_main]
+#![no_std]
+
+// Pick one of these panic handlers:
+
+// `panic!` halts execution; the panic message is ignored
```

Comment:
I don't think it's ignored, right? It should be printed via semihosting.

```suggestion
```

Reply:
ah, I was removing some of the original code and missed removing these comments, thanks